Header header

# Velocity and Heading
float64 VelN
float64 VelE
float64 VelD

float64 speed

float64 heading